statics/js/components/object-detail.js

```diff
@@ -69,7 +69,7 @@ Vue.component('object-detail', {
 
     collapsedByDefault: function(key) {
       if (key === "FDB" || key === "Neighbors" || key === "RoutingTable" ||
-          key === "CPU" || key === "IsolatedCPU")
+          key === "CPU" || key === "IsolatedCPU" || key == "Sockets")
```

Comment:
nothing related to this line but I think we need to remove the option for SocketInfo at lhe capture creation level

Reply:
done